src/qt/choice.cpp

```diff
@@ -20,6 +20,8 @@ namespace
 class LexicalSortProxyModel : public QSortFilterProxyModel
 {
 public:
+    LexicalSortProxyModel(QObject* owner) : QSortFilterProxyModel(owner) {}
```

Comment:
```suggestion
    explicit LexicalSortProxyModel(QObject* owner) : QSortFilterProxyModel(owner) {}
```

Reply:
Just out of curiosity, what implicit possible conversion will that prevent?

Reply:
Well, any Qt object (they all derive from `QObject`, right?) could be passed to a function taking `LexicalSortProxyModel` accidentally. I don't know how likely is it, but why even spend time thinking about it when we know that we _never_ want any implicit conversions here?